cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -114,6 +117,7 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         cy.wait("@getApplication");
         cy.wait(2 * SEC);
         // No credentials required for uploaded binary.
+        uploadBinaryApplication.perform_assessment("low", [stakeHolder]);
```

Comment:
Could you add this check to verify assessment completed wherever assessment is being performed in this file?
uploadBinaryApplication.verifyStatus("assessment", "Completed");

Reply:
I will, @nachandr , not a big deal

Reply:
Done